README.md

```diff
@@ -276,6 +276,7 @@ See the [API documentation](./docs/api.md) for all endpoints.
 - [gptel Emacs client](https://github.com/karthink/gptel)
 - [Oatmeal](https://github.com/dustinblackman/oatmeal)
 - [cmdh](https://github.com/pgibler/cmdh)
+- [Nano Bots CLI](https://github.com/icebaker/ruby-nano-bots)
```

Comment:
This is super cool! Would it be possible to only add this once just since it's the same URL in both cases